src/NServiceBus.Core/Licensing/LicenseManager.cs

```diff
@@ -2,37 +2,81 @@ namespace NServiceBus
 {
     using System;
     using System.Diagnostics;
-#if NETSTANDARD
-    using System.Runtime.InteropServices;
-#endif
     using System.Text;
     using System.Threading;
     using System.Threading.Tasks;
     using Logging;
     using Particular.Licensing;
+#if NETSTANDARD
+    using System.Runtime.InteropServices;
+
+#endif
```

Comment:
Why has this moved? The original place seems correct.

Reply:
good point, not sure why it was moved. I did a reformat of the file but I can revert that change.